client/account-recovery/reset-password/reset-password-email-form/index.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+
+const ResetPasswordEmailSent = ( props ) => {
+	const {
+		// TODO: uncomment it once the real data got wired
+		// email,
+		translate,
+	} = props;
+
+	return (
+		<div className="reset-password-email-form">
+			<h2 className="reset-password-email-form__title">
+				{ translate( 'Reset your password' ) }
+			</h2>
+			<Card>
+			<p>
+				{ translate(
+					'We have sent an email to {{strong}}%(email)s{{/strong}} with a reset link. ' +
+					'Click on it to reset your password and recover access to your account.',
+					{
+						components: { strong: <strong /> },
+						// TODO: it will become args: { email } when the real data got wired in the up-coming PR.
+						// args: { email }
```

Comment:
Ah, nice tip :)
The default prop is now added by f3d6a75f94984b3d06eba1666122b37dd3947c97